Update-converter-nue-to-enu

joachimhgg opened the pull request:
**Description**

Fix road conversion , improve general robustness, improve description

BenjaminDeleze reviewed (changes requested):
The script works really well in general but I saw some mistakes in the conversion for:
- [x] crossroads
- [x] plane and floor
- [x] stairs (in `spot.wbt`)
- [x] radiator, officeChair, cabinet, stairs (in `hall.wbt`)
- [ ] sofa (in `nao_room`)

The support for elevation_grid is missing.

I dont think everything need necessary to be fixed, it is something we can discuss...

I think it is important to explain in the wiki that it will only work for worlds that are in NUE. Worlds in ENU will be broken but I see no solution.

joachimhgg commented on scripts/converter/convert_nue_to_enu_rub_to_flu.py:
```diff
@@ -310,11 +330,12 @@ def convert_nue_to_enu_world(filename, mode='all', objects_pi=[], objects_pi_2=[
     # possibility to use an argv, a list or a folder
-    filename_list = []  # example: 'projects/robots/robotcub/icub/worlds/icub_stand.wbt', change it by your .wbt or.proto
+    # example: 'projects/robots/robotcub/icub/worlds/icub_stand.wbt', change it by your .wbt or.proto
+    filename_list = ['projects/robots/softbank/nao/protos/Nao.proto']
```
No, I am removing it.

joachimhgg commented:
> The script works really well in general but I saw some mistakes in the conversion for:
> 
>     * [x]  crossroads
> 
>     * [x]  plane and floor
> 
>     * [x]  stairs (in `spot.wbt`)
> 
>     * [x]  radiator, officeChair, cabinet, stairs (in `hall.wbt`)
> 
>     * [x]  sofa (in `nao_room`)
> 
> 
> The support for elevation_grid is missing.
> 
> I dont think everything need necessary to be fixed, it is something we can discuss...
> 
> I think it is important to explain in the wiki that it will only work for worlds that are in NUE. Worlds in ENU will be broken but I see no solution.

I fixed the crossroad, viewpoint and added objects into the lists. For the `ElevationGrid`, it is too much time to do it so let's wait if there is a need for it, maybe we could share your script as well.

BenjaminDeleze reviewed (approved):
Thank you!
